.github/workflows/release.yml

```diff
@@ -5,6 +5,7 @@ on:
       - master
 jobs:
   release_bin:
+    if: "${{ startsWith(github.event.head_commit.message, 'chore(server): release v') == true }}"
```

Comment:
It basically means that we only create a new release when the last commit message on master starts with `chore(server): release v`. Note that only the Create Release Pull Request workflow creates those commits.